tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -55,6 +55,16 @@ def delete(self, group_id):
         self.delete_request(url)
         logger.info('Deleted single group (ID: {0})'.format(group_id))
 
+    @api(version="2.0")
+    def update(self, group_item, default_site_role=UserItem.Roles.Unlicensed):
```

Comment:
Pull `UserItem.Roles.Unlicensed` into a constant `UNLICENSED_USER` or something a little shorter, it helps this read a little easier for me